### PR TITLE
Cleaned up the logging in to dashboard part.

### PR DESCRIPTION
**Pull request by sguerciojr**
removed the 2fa screenshot
cleaned up the wording.

### CODE REVIEW

File: pages/cloud/using.md [hunk 1]
```diff
@@ -2,25 +2,19 @@
 
 ### Managing your Project with the Cloud Dashboard
 
-Once you've requested and been granted an account on the LakeEffect cloud, you will be able to login in to the web console.
-Your username and password for LakeEffect are the same as your CCR username and password.  However, you need additional 
-privileges to be allowed to login to the cloud.  You will be informed when your cloud account is ready.
+Allocations to the LakeEffect Cloud are requested and managed through the [ColdFront Resource Allocations Management Tool](../portals/coldfront.md).
```
Comment (aebruno) [inline pages/cloud/using.md:5]: Can you link to getting-access.md#allocation-requests

So this should be: 

To request an allocation to LakeEffect [see here](getting-access.md#allocation-requests)

File: pages/cloud/using.md [hunk 2]
```diff
@@ -2,25 +2,19 @@
...
+From the [The LakeEffect Web Console](https://dashboard.cloud.ccr.buffalo.edu), you will need to authenticate using the CCR OpenID Connect option with your CCR username, password and otp.
 
-You will need to authenticate to CCR's **Identity Management System** with your CCR username, password and otp.
-For more information about logging in with two-factor authentication enabled click [here](../2fa.md).
-
-![](../images/cloud/ccridm2.png)
+![](../images/cloud/openid.png)
 
-[More information on two factor authentication](../2fa.md)
+For more information about logging in with two-factor authentication enabled click [here](../getting-access.md).
```
Comment (aebruno) [inline pages/cloud/using.md:17]: I would just completely remove this.

File: pages/cloud/using.md [hunk 3]
```diff
@@ -2,25 +2,19 @@
...
 
 !!! Warning
     You must be on the UB campus or connected to the [UB VPN](https://buffalo.edu/ubit) from off campus in order to access CCR's portals
 
-[The LakeEffect Dashboard (Horizon)](https://dashboard.cloud.ccr.buffalo.edu) - simplifies management of your project's cloud resources and provides information on your group's usage.
 
-![](../images/cloud/openid.png)
-
-Two factor authentication must be enabled on your CCR account in order to access CCR's LakeEffect Dashboard.  If you do not, you will get the error ``You don't have access to this resource`` when attempting to login.
+From the [The LakeEffect Web Console](https://dashboard.cloud.ccr.buffalo.edu), you will need to authenticate using the CCR OpenID Connect option with your CCR username, password and otp.
```
Comment (aebruno) [inline pages/cloud/using.md:13]: Perhaps here we add link to getting-access.md..


you will need to authenticate using your CCR account. Don't have an account? [see here](getting-access.md)

**Review by aebruno — CHANGES_REQUESTED** [review]

File: pages/cloud/using.md (hunk 1 above)
Comment (sguerciojr) [reply pages/cloud/using.md:5]: done

File: pages/cloud/using.md (hunk 2 above)
Comment (sguerciojr) [reply pages/cloud/using.md:17]: removed

File: pages/cloud/using.md (hunk 3 above)
Comment (sguerciojr) [reply pages/cloud/using.md:13]: Want to tell people they need to pick Open id versus keystone
How about:

From the [The LakeEffect Web Console](https://dashboard.cloud.ccr.buffalo.edu), select CCR OpenID Connect option and authenticate using your CCR account. Don't have an account? [see here](../getting-access.md)

**Review by aebruno — APPROVED** [review]

**Pull request merged**